frontend/src/components/LineInputList.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
-
-    export let values;
+    export let values: string[];
+    export let placeholder: string;
```

Comment:
I have this in my console:
```
 "Component has unused export property 'placeholder'. If it is for external reference only, please consider using `export const placeholder`"
```
Do you as well?

Reply:
working on it. I must have missed adding some changes.

Reply:
I think you fixed it. Not appearing anymore.